nntrainer/tensor/blas_interface.cpp

```diff
@@ -659,6 +659,16 @@ static unsigned int isamax_raw(const unsigned int N, const float *X,
 
 #endif
 
+void transpose_matrix(const unsigned int M, const unsigned int N,
+                      const float *src, unsigned int ld_src, float *dst,
+                      unsigned int ld_dst) {
+#ifdef USE_NEON
+  transpose_neon<float>(M, N, src, ld_src, dst, ld_dst);
+#else
+  transpose_fallback<float>(M, N, src, ld_src, dst, ld_dst);
+#endif
+}
```

Comment:
Someday, all of such #ifdef (NEON, FP16, ...) need to be migrated to header files (preferably centralized into a single header) and code lines depending on such need to be separated as a class or a file (e.g., defined in ```blas_neon.cpp``` and whether to use functions/sub-classes in that file is determined at a header and build-script/option.

But for today, lets move on.

Reply:
Seems this issue is related to #2549
I am aware of it. Think we urgently have to discuss when to apply this

Reply:
In fact, one of the biggest problems of applying #2549 is that all android make file / tizen spec file in the current nntr is including `blas_interface.h` since this pr substitutes `blas_interface` into single `cpu_backend.h`, this affects quite a lot of files...